config/README.md

```diff
@@ -15,7 +15,7 @@ Getting started
 
 1. [Add Firebase to your Android Project](https://firebase.google.com/docs/android/setup).
 2. Create a service account as described in [Adding Firebase to your Server](https://firebase.google.com/docs/admin/setup) and download the JSON file.
-  - Copy the private key JSON file to this folder and rename it to `service-account.json`.
+  - Set `GOOGLE_APPLICATION_CREDENTIALS` environment variable to path of downloaded credential file.
```

Comment:
Maybe show how you can use `gcloud auth login default` (or whatever it is) to get it working without even having to download the JSON file?

Reply:
I'm curious about that too.  The best practice is to set that var if you're running locally or in a non-Google server environment, correct?  But this *would* work in GCP/GAE without the var properly set, right Arthur?

Reply:
Even locally I never set the environment variable unless I am trying to run two different things with different service accounts.  Normally the default machine-wide credential Just Works :tm:

Reply:
The `gcloud auth application-default login` command does not work with some of the Firebase APIs so lets go with the env var for now and change later on when/if gcloud gets wider support.